### DISCUSSION TITLE
Can we directly upgrade argorollouts of version 1.1.1 to 1.2.3 by apply yaml of 1.2.3?

### DISCUSSION THREAD

**Discussion by saiharshitach** [argoproj/argo-rollouts#2423] [Q&A]
Can we directly upgrade argorollouts of version 1.1.1 to 1.2.3 by apply yaml of 1.2.3?

What is the upgrade process for argorollouts component?Does it impatc existing objects of rollouts in any way during this upgrade?

Thank you

**Comment by kostis-codefresh** [comment]
Check https://argoproj.github.io/argo-rollouts/installation/#upgrading-argo-rollouts
